evap/staff/views.py

```diff
@@ -2263,6 +2265,27 @@ def faq_section(request, section_id):
     return render(request, "staff_faq_section.html", template_data)
 
 
+@manager_required
+def infotexts(request):
+    InfoTextFormSet = modelformset_factory(Infotext, form=InfotextForm, edit_only=True, extra=0)
+    formset = InfoTextFormSet(request.POST or None, queryset=Infotext.objects.all())
+
+    if formset.is_valid():
+        formset.save()
+        messages.success(request, _("Successfully updated the infotext entries."))
+        return redirect("staff:infotexts")
+    if formset.errors:
+        messages.error(
+            request, _("The form was not saved and no infotexts were changed. Please resolve the errors shown below.")
+        )
```

Comment:
I don't think we manually add an error message in any other place, we always get the automatic one by django. Why does that not work here?